dll/win32/shell32/wine/classes.c

```diff
@@ -130,9 +130,11 @@ BOOL HCR_MapTypeToValueA(LPCSTR szExtension, LPSTR szFileType, LONG len, BOOL bP
 	return TRUE;
 }
 
+EXTERN_C HRESULT SHELL32_EnumDefaultVerbList(LPCWSTR List, UINT Index, LPWSTR Verb, SIZE_T cchMax);
```

Comment:
It looks like all your changes in this file need to be inside a `#ifdef __REACTOS__ ... #endif` block.